packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -66,14 +74,26 @@ export default {
       hidden: !!this.link.hidden
     };
   },
+  computed:{
+    isLink: function(){
```

Comment:
Would be better if these were enclosed as a property named like `entityType` and computed as a single function so we guarantee only one can happen at once